docs/rules/replace-expected-exception.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Replace JUnit ExpectedException Rule with assertThrows
+ruleId: ReplaceExpectedException
+since: 3.24.0
+minJavaVersion: 8
+remediationCost: 5
+    
+description:
+    The goal of this rule is to replace the JUnit ExpectedException with 'assertThrows'. 
+tags: ["Java 8", "Testing", "Lambda", "Readability"]
+---
+
+# Replace JUnit ExpectedException Rule with assertThrows
+
+[[toc]]
+
+## Properties
+
+<RuleProperties />
+
+## Description
+
+The [`ExpectedException.none()`](https://junit.org/junit4/javadoc/latest/org/junit/rules/ExpectedException.html#none()) is deprecated since deprecated since JUnit 4.13. 
+The recommended alternative is to use [`assertThrows()`](). 
+This makes JUnit tests easier to understand and prevents the scenarios where some parts the test code is unreachable. 
```

Comment:
Remove one "the", add one "of", and use "are" instead of "is". The sentence should be:

This makes JUnit tests easier to understand and prevents scenarios where some parts of the test code are unreachable.